### PR TITLE
Website Updates

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
Website Updates

### CODE REVIEW

File: docs/src/pages/index.js [hunk 1]
```diff
@@ -24,17 +23,45 @@ function Home() {
...
-              </code></pre></div>    </div>
-                  </li>
-                <li>When ready, open and pull request and request a review from a contributor.</li>
-              </ul>
-            </section>
+            <div className={styles.section}>
+                <h1>Why Birb</h1>
+                <div>
+                  <h2>Easy to learn</h2>
+                  <p>Eaay to make apps in less amount of code and pretty easy to learn if you know any other language</p>
```
Comment (Calamity210) [inline docs/src/pages/index.js:30]: Easy to make apps?

File: docs/src/pages/index.js [hunk 2]
```diff
@@ -24,17 +23,45 @@ function Home() {
...
+                  <img src="img/code.svg" height="300px" color="#6ac00d"></img>
+                </div>
+                <div>
+                  <h2>Null Saftey</h2>
```
Comment (Calamity210) [inline docs/src/pages/index.js:34]: It would be spelled safety

File: docs/src/pages/index.js [hunk 3]
```diff
@@ -24,17 +23,45 @@ function Home() {
...
+                  <p>Avoid entire classes of errors such as null pointer exceptions</p>
```
Comment (Calamity210) [inline docs/src/pages/index.js:35]: You wouldnt encounter a null pointer in dart nor in birb regardless, and null safety isnt really ready so I wouldn't put this up yet

File: docs/src/pages/index.js [hunk 4]
```diff
@@ -24,17 +23,45 @@ function Home() {
...
+                  <img src="img/null-saftey.svg" height="300px" color="#6ac00d"></img>
+                </div>
+                <div>
+                  <h2>Tool Friendly</h2>
+                  <p>Program in Birb using the tools that you already use</p>
+                  <div className={styles.Row}>
+                    <div className={styles.Card}>
+                      <img src="img/vscode.png" height="50px"></img>
+                      <p>Install Birb Code VS Code extention from VS Code Marketplace</p>
```
Comment (Calamity210) [inline docs/src/pages/index.js:44]: Extension

File: docs/src/pages/index.js [hunk 5]
```diff
@@ -24,17 +23,45 @@ function Home() {
...
+                      </div>
+                    </div>
+                    <div className={styles.Card}>
+                      <img src="img/terminal.png" height="50px"></img>
+                      <p>Use the interactive Birb shell in your terminal / bash</p>
+                    </div>
+                  </div>
+                </div>
+            </div>
+            <h1 align="center">Try Birb in your browser</h1>
```
Comment (Calamity210) [inline docs/src/pages/index.js:63]: Repeated phrase, try rephrasing it

**Review by Calamity210 — CHANGES_REQUESTED** [review]

**Comment by Calamity210** [comment]
@Andy-Python-Programmer can you provide a preview of what the changes actually look like?

**Comment by Andy-Python-Programmer** [comment]
EC414DA6-2B52-4049-9409-A077067353EB-1436-000001ACFA0A62C5
F838787E-D868-4B78-B9B7-01E741C5FA47-1436-000001ACFE73A60F

@Calamity210 The image is a bit blurry as I just pulled them from Discord. But this is exact design how they will look.

File: docs/src/pages/index.js [hunk 6]
```diff
@@ -24,17 +23,40 @@ function Home() {
...
+                    <div className={styles.Card}>
+                      <img src="img/vscode.png" height="50px"></img>
+                      <p>Install Birb Code VS Code Extension from VS Code Marketplace</p>
+                      <div className={styles.buttons}>
+                        <Link to={"https://marketplace.visualstudio.com/items?itemName=BirbDev.birbcode"}>Extension</Link>
+                      </div>
+                    </div>
+                    <div className={styles.Card}>
+                      <img src="img/logo.png" height="50px"></img>
+                      <p>Try Birb in your browser! All of the Birb core libs are included</p>
```
Comment (Calamity210) [inline docs/src/pages/index.js:46]: Prefer to not use abbreviations

File: docs/src/pages/index.js [hunk 7]
```diff
@@ -24,17 +23,40 @@ function Home() {
...
+                      <div className={styles.buttons}>
+                        <Link to={"https://play-birbolang.web.app/#/"}>Playground</Link>
+                      </div>
+                    </div>
+                    <div className={styles.Card}>
+                      <img src="img/terminal.png" height="50px"></img>
+                      <p>Use the interactive Birb shell in your terminal / bash</p>
```
Comment (Calamity210) [inline docs/src/pages/index.js:53]: / bash is not needed

File: docs/src/pages/index.js [hunk 8]
```diff
@@ -24,17 +23,40 @@ function Home() {
...
+                    </div>
+                  </div>
+                </div>
+            </div>
+            <h1 align="center">Want to try Birb? Give it a shot!</h1>
+            <iframe src="https://play-birbolang.web.app/#/" width="95%" height="500px" scrolling="yes" align="center"></iframe>
```
Comment (Calamity210) [inline docs/src/pages/index.js:59]: Can you center this horizontally?

**Review by Calamity210 — CHANGES_REQUESTED** [review]

File: docs/src/pages/index.js (hunk 7 above)
Comment (Andy-Python-Programmer) [reply docs/src/pages/index.js:53]: Yeup

**Pull request closed without merging**